source/common/network/io_uring_socket_handle_impl.h

```diff
@@ -8,6 +8,10 @@
 #include "source/common/common/logger.h"
 #include "source/common/network/io_socket_handle_base_impl.h"
 
+#if defined(__ANDROID_API__)
+#include "does_not_exist.h"
```

Comment:
nice, so effectively if this somehow leaks into mobile we get a compile-time error, correct?

Reply:
Yes, we don't want io_uring socket handle in Android.